CHANGELOG.md

```diff
@@ -5,6 +5,11 @@ in each resource release, will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
+## [v5.11.0.11] - 2022-01-20
+
+### Changed
+- Changed base image of dockerfiles to an OS image and installed relevant jdk on it, instead of directly using a JDK pre installed base image. 
```

Comment:
instead of directly using a JDK pre installed base image.   -> instead of using JDK install OS image.